plugins/org.polarsys.capella.transition.system2subsystem.crossphases/src/org/polarsys/capella/transition/system2subsystem/crossphases/rules/fa/FunctionalExchangeRule.java

```diff
@@ -20,10 +20,10 @@
 import org.eclipse.core.runtime.Status;
 import org.eclipse.emf.common.util.EList;
 import org.eclipse.emf.ecore.EObject;
-import org.polarsys.capella.core.data.capellacore.CapellaElement;
+import org.polarsys.capella.common.queries.AbstractQuery;
+import org.polarsys.capella.core.business.queries.queries.fa.GetAvailable_ComponentExchange_AllocatedFunctionalExchanges;

```

Comment:
This class is implementation, 
https://github.com/eclipse/capella-sss-transition/commit/4489bb893ef3f2da6306c1a999261f80a4ac0747#diff-e1bf1fe7057e20660b63778e5e5712b3b0193bb3ab524d82c4de0e8b4f5cec63R24

shall not be instanciated directly,

BusinessQueriesProvider.getContribution(EClass cls, EStructuralFeature feature).getAvailableElements()
or through its constant.
https://github.com/eclipse/capella/blob/409a90c4a2ed6c9c355badee0d068c32ec47f195/core/plugins/org.polarsys.capella.core.data.business.queries/src/org/polarsys/capella/core/business/queries/fa/ComponentExchange_AllocatedFunctionalExchanges.java#L49